### PR TITLE
Add first version

### PR DESCRIPTION
**Pull request by Evnsan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
...
+
+usage: kafka-offset-migrate [-h] [-s SRC_KAFKA] [-d DST_KAFKA] [-i SRC_ZK]
+                            [-o DST_ZK] [-t TOPIC] [-g GROUP]
+                            {k2k,z2z,z2k} ...
+
+Simple tool to migrate offsets between kafkas.
+
+positional arguments:
+  {k2k,z2z,z2k}         sub-command help
+    k2k                 k2k will fetch offsetsi from src kafka and set then
```
Comment (stoiev) [inline README.md:22]:
```suggestion
    k2k                 k2k will fetch offsets from src kafka and set then
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,43 @@
 # kafka-offset-migrate
 This is a simple tool to migrate consumer offtsets from a kafka cluster to a another.
```
Comment (stoiev) [inline README.md:2]:
```suggestion
This is a simple tool to migrate consumer offtsets from a kafka cluster to another.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,43 @@
...
+                        into dstkafka.
+    z2z                 z2z will fetch offsets from src zookeeper and set then
+                        into dstzookeeper.
+    z2k                 z2k will fetch offsets from src zookeeper and set then
+                        into dst kafka.
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -s SRC_KAFKA, --src-kafka SRC_KAFKA
+                        The kafka adresse list from where get ofssets.
```
Comment (stoiev) [inline README.md:32]: `ofssets` and `adresse` are typos found in some places. Commenting only this one 
```suggestion
                        The kafka address list from where get offsets.
```

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    author_email='evandro@cobli.co',
+    license='MIT',
+    install_requires=[
+        'kafka-python==1.4.6',
+        'kazoo==2.6.1'
+    ],
+    entry_points={
+        'console_scripts': ['kafka-offset-migrate=kafka_offset_migrate.main:main']
+    },
+keywords='kakfas offset migrate')
```
Comment (stoiev) [inline setup.py:22]:
```suggestion
    },
    keywords='kafkas offset migrate')
```

File: kafka_offset_migrate/kafka_client.py [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+            tp: kafka.OffsetAndMetadata(offset.offset, b'') for
+            tp, offset in kafka_offsets.items()}
+
+        print(offset_and_metadata)
+
+        consumer.topics()
+        consumer.commit(offset_and_metadata)
+        consumer.close()
+
+    def kafka_to_kakfa(self):
```
Comment (stoiev) [inline kafka_offset_migrate/kafka_client.py:155]:
```suggestion
    def kafka_to_kafka(self):
```

File: kafka_offset_migrate/kafka_client.py [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+        old_zk_offsets = self.get_offsets_from_zk(self.path, self.src_zk_hosts)
+        # Convert Offsets to timestamp
+        old_kafka_offsets = self.zk_offsets_to_kafka_offsets(old_zk_offsets,
+                                                             self.topic)
+        if False:
+            timestamps = self.convert_offsets_to_timestamp(
+                self.topic, old_kafka_offsets, self.src_kafka_hosts)
+        # Get offsets from dst kafka
+            new_kafka_offsets = self.convert_timestamp_to_offsets(
+                self.group_id, self.topic, timestamps, self.dst_kafka_hosts)
```
Comment (stoiev) [inline kafka_offset_migrate/kafka_client.py:197]: Unreachable code. Maybe debug stuff?

File: kafka_offset_migrate/main.py [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+    dst_kafka_hosts = getattr(opts, 'dst_kafka', None)
+    src_zk_hosts = getattr(opts, 'src_zk', None)
+    dst_zk_hosts = getattr(opts, 'dst_zk', None)
+    group_id = getattr(opts, 'group', None)
+    topic = getattr(opts, 'topic', None)
+
+    client = KafkaClient(group_id, topic, src_kafka_hosts, dst_kafka_hosts,
+                     src_zk_hosts, dst_zk_hosts)
+    if opts.action == 'k2k':
+        client.kafka_to_kakfa()
```
Comment (stoiev) [inline kafka_offset_migrate/main.py:98]:
```suggestion
        client.kafka_to_kafka()
```

File: kafka_offset_migrate/kafka_client.py (hunk 6 above)
Comment (Evnsan) [reply kafka_offset_migrate/kafka_client.py]: This code doesn't make sense Oo

File: kafka_offset_migrate/main.py [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+    parser.add_argument('-g', '--group', default=None,
+                        help='Group that owns the offset.')
+
+    cmds = parser.add_subparsers(help='sub-command help')
+
+
+    ## k2k
+    kafka_to_kafka_cmd = cmds.add_parser('k2k',
+                                         help='k2k will fetch offsetsi from '
```
Comment (stoiev) [inline kafka_offset_migrate/main.py:40]:
```suggestion
                                         help='k2k will fetch offsets from '
```

File: kafka_offset_migrate/main.py [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+def parse_command_line():
+    parser = argparse.ArgumentParser(description='Simple tool to migrate '
+                                                 'offsets between kafkas.')
+
+    parser.add_argument('-s', '--src-kafka', default='localhost:9092',
+                        help='The kafka adresse list from where get ofssets.')
```
Comment (stoiev) [inline kafka_offset_migrate/main.py:16]: @Evnsan, some entries here have the same typos that was on readme, and there is more of this in readme. Maybe it's better a search/replace on project.
```suggestion
                        help='The kafka adress list from where get offsets.')
```

File: setup.py [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+    author_email='evandro@cobli.co',
+    license='MIT',
+    install_requires=[
+        'kafka-python==1.4.6',
+        'kazoo==2.6.1'
+    ],
+    entry_points={
+        'console_scripts': ['kafka-offset-migrate=kafka_offset_migrate.main:main']
+    },
+keywords='kafkas offset migrate')
```
Comment (stoiev) [inline setup.py:22]:
```suggestion
    },
    keywords='kafkas offset migrate')
```

File: README.md [hunk 11]
```diff
@@ -1,2 +1,43 @@
 # kafka-offset-migrate
-This is a simple tool to migrate consumer offtsets from a kafka cluster to a another.
+This is a simple tool to migrate consumer offtsets from a kafka cluster to another.
+
+In order to migrate a consumer group GROUP of a topic TOPIC you must stop any client
+consuming in this group and run:
+```bash
+$ kafka-offset-migrate k-2-k -s $SOURCE_KAFKA -d $DESTINATION_KAFKA -g $GROUP -t $TOPIC
```
Comment (stoiev) [inline README.md:7]:
```suggestion
$ kafka-offset-migrate k2k -s $SOURCE_KAFKA -d $DESTINATION_KAFKA -g $GROUP -t $TOPIC
```

File: setup.py (hunk 4 above)
Comment (Evnsan) [reply setup.py]: Done

File: kafka_offset_migrate/main.py (hunk 8 above)
Comment (Evnsan) [reply kafka_offset_migrate/main.py]: Done

File: kafka_offset_migrate/main.py (hunk 9 above)
Comment (Evnsan) [reply kafka_offset_migrate/main.py]: Done

File: README.md (hunk 11 above)
Comment (Evnsan) [reply README.md]: Done

**Pull request merged**